person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(id, age, parent_permission: true, name: 'Unknown')
+    super(id, age)
```

Comment:
- [ ] Kindly ensure you run your code and be sure it works. There was an error when testing your app due the super method here. Kindly remove this `id` and `age` from the super arguments. You should have something like this:

  ```
  super()
  ```